templates/partials/question.html

```diff
@@ -91,10 +91,10 @@
   "instruction": question_instruction,
   "legendIsQuestionTitle": should_wrap_with_fieldset(question)
 }) %}
-  {%- if content.list -%}
+  {%- if content.list and content.list.list_items and content.list.list_items | length > 0 -%}
```

Comment:
Not sure why you have checked for the existence of content.list.list_items, then its length? content.list.list_items would just return false if empty so you don't need the length check